README.md

```diff
@@ -214,6 +215,7 @@ statement instead the previous block.
 | <a name="input_cd_authorization_policy_creation"></a> [cd\_authorization\_policy\_creation](#input\_cd\_authorization\_policy\_creation) | Disable Toolchain Service to Secrets Manager/Key Protect/Notifications Service authorization policy creation. To disable set the value to `disabled`. | `string` | `""` | no |
 | <a name="input_cd_change_management_group"></a> [cd\_change\_management\_group](#input\_cd\_change\_management\_group) | Specify group for change management repository | `string` | `""` | no |
 | <a name="input_cd_change_management_repo_auth_type"></a> [cd\_change\_management\_repo\_auth\_type](#input\_cd\_change\_management\_repo\_auth\_type) | Select the method of authentication that is used to access the Git repository. Valid values are 'oauth' or 'pat'. Defaults to `oauth` when unset. `pat` is a git `personal access token`. | `string` | `""` | no |
+| <a name="input_cd_change_management_repo_git_provider"></a> [cd\_change\_management\_repo\_git\_provider](#input\_cd\_change\_management\_repo\_git\_provider) | Git provider for the change management repo. If not set will default `hostedgit`. | `string` | `""` | no |
```

Comment:
Git provider for the change management repo. If not set will default to `hostedgit`.